docs/book/src/tasks/verify-container-images.md

```diff
@@ -0,0 +1,54 @@
+# Verification of CAPI artifacts
+
+## Requirements 
+You will need to have the following tools installed:
+- cosign [(install guide)](https://docs.sigstore.dev/system_config/installation/)
+- jq [(download jq)](https://stedolan.github.io/jq/download/)
+
+## CAPI Images
+Each [release](https://github.com/kubernetes-sigs/cluster-api/releases) of the Cluster API project includes the following container images:
+- cluster-api-controller
+- kubeadm-bootstrap-controller
+- kubeadm-control-plane-controller
+- clusterctl
+
+## Verifying Image Signatures
+All of the four images are hosted by [registry.k8s.io](https://registry.k8s.io). In order to verify the authenticity of the images, you can use `cosign verify` command with the appropriate image name and version: 
+
+```bash
+$ cosign verify registry.k8s.io/cluster-api/cluster-api-controller:v1.5.0 --certificate-identity krel-trust@k8s-releng-prod.iam.gserviceaccount.com --certificate-oidc-issuer https://accounts.google.com | jq .
+
```

Comment:
Can we split this command and the output in separate code blocks? Just to make it possible to use the copy button for the cmd in the book